cpp/ql/src/semmle/code/cpp/models/interfaces/Taint.qll

```diff
@@ -15,11 +15,14 @@ import semmle.code.cpp.models.Models
  * A library function for which a taint-tracking library should propagate taint
  * from a parameter or qualifier to an output buffer, return value, or qualifier.
  *
+ * An expression is tainted if it could be influenced by an attacker to have
+ * an unusual value.
+ *
```

Comment:
I know @jbj mentioned this the other day as his definition of "taint", but he also said that this changes every week. Do we really want to put this definition in a public qldoc?

Reply:
It's only every month, not every week ;-). I've [asked on Slack](https://github.slack.com/archives/CP6990YTU/p1590581079051800) to see if the language teams can agree on a definition.

Reply:
I'm keen for us to document a working definition of taint, even if we decide to change it later.  It seems better than everyone inventing their own understanding, or trying to infer it backwards from existing behaviour of the models (as I have been).